acquisition.py

```diff
@@ -86,7 +98,97 @@ def get_reward_batch(self, inputs_af_base):  # inputs_af = list of ...
         inputs = pad_sequence(inputs_af, batch_first=True, padding_value=0.0)
 
         self.load_best_proxy()
-        self.proxy.model.eval()
+        self.proxy.model.train()
         with torch.no_grad():
             outputs = self.proxy.model(inputs, input_afs_lens, None)
         return outputs
+
+
+class AcquisitionFunctionOracle(AcquisitionFunctionBase):
+    def __init__(self, config, proxy):
+        super().__init__(config, proxy)
+
+    def get_reward_batch(self, inputs_af_base):
+        super().get_reward_batch(inputs_af_base)
+        outputs = self.proxy.get_score(inputs_af_base)
```

Comment:
Why do we call the proxy here ? We should just call the oracle no ? 
Unless you have indicated that the proxy is the oracle in the gfn vanilla setting...
For clarity, I would suggest sticking with calling the oracle in the name function.
If the acquisition function only depends on the proxy (like for the other classic Acquisition Function), we can implement in the proxy class named get_oracle_score, but then the proxy class would depend on the oracle class...
Organization to be discussed !

Reply:
Yeah proxy is the oracle in the vanilla setting. I did it so that the name remains consistent through the acquisition class (must be so else line 17 in this file would give an error if called with oracle). 
Yeah let's discuss this, not sure I completely understand your sugfestion